README.md

```diff
@@ -286,6 +286,9 @@ By default, auto-cpufreq does not use the config file! If you wish to use it, th
 # preferred governor
 governor = performance
 
+# see available preferences by running: cat /sys/devices/system/cpu/cpu0/cpufreq/energy_performance_preference
```

Comment:
This is incorrect, if you want to see available preferences you should do:
`cat /sys/devices/system/cpu/cpu0/cpufreq/energy_performance_available_preferences`

Please update for every other comment where:

```
# see available preferences by running: cat /sys/devices/system/cpu/cpu0/cpufreq/energy_performance_preference
```

is mentioned.